[2024-03-20T04:01:50.637+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_witth_cathup_backfil_v4.bash_task manual__2024-03-20T04:01:49.950820+00:00 [queued]>
[2024-03-20T04:01:50.641+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_witth_cathup_backfil_v4.bash_task manual__2024-03-20T04:01:49.950820+00:00 [queued]>
[2024-03-20T04:01:50.641+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-20T04:01:50.646+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): bash_task> on 2024-03-20 04:01:49.950820+00:00
[2024-03-20T04:01:50.648+0000] {standard_task_runner.py:60} INFO - Started process 9529 to run task
[2024-03-20T04:01:50.650+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_witth_cathup_backfil_v4', 'bash_task', 'manual__2024-03-20T04:01:49.950820+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_cathup_backfil.py', '--cfg-path', '/tmp/tmpiii4dlzh']
[2024-03-20T04:01:50.651+0000] {standard_task_runner.py:88} INFO - Job 45: Subtask bash_task
[2024-03-20T04:01:50.673+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_witth_cathup_backfil_v4.bash_task manual__2024-03-20T04:01:49.950820+00:00 [running]> on host be400aaaf05d
[2024-03-20T04:01:50.705+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Hitesh' AIRFLOW_CTX_DAG_ID='dag_witth_cathup_backfil_v4' AIRFLOW_CTX_TASK_ID='bash_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-20T04:01:49.950820+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-20T04:01:49.950820+00:00'
[2024-03-20T04:01:50.706+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-20T04:01:50.707+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world!']
[2024-03-20T04:01:50.711+0000] {subprocess.py:86} INFO - Output:
[2024-03-20T04:01:50.712+0000] {subprocess.py:93} INFO - hello world!
[2024-03-20T04:01:50.712+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-20T04:01:50.722+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=dag_witth_cathup_backfil_v4, task_id=bash_task, execution_date=20240320T040149, start_date=20240320T040150, end_date=20240320T040150
[2024-03-20T04:01:50.748+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-20T04:01:50.755+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
